low-poly-world/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="initial-scale=1" />
     <title>Low poly map of the world</title>
-    <link rel="stylesheet" href="https://js.arcgis.com/4.24/esri/css/main.css" />
+    <link rel="stylesheet" href="https://js.arcgis.com/4.23/esri/css/main.css" />
```

Comment:
This one I actually downgraded because it broke with 4.24 from a breaking change, see issue https://github.com/RalucaNicola/JSAPI_demos/issues/21.